src/analyze/analysis_handler.go

```diff
@@ -10,22 +10,41 @@ import (
 )
 
 // Takes in a Window, parses object into frequency counts, and then calls specified analysis functions
+//code to write the frequencies; the outliers; and the minReqs to files
 func AnalyzeBGPMessages(window common.Window) {
 	bucketMap := window.BucketMap
 
 	// Convert BucketMap to a map of timestamp to length of messages
 	lengthMap := make(map[time.Time]float64)
+	timeStampsFull := []string{}
 
 	for timestamp, messages := range bucketMap {
 		lengthMap[timestamp] = float64(len(messages))
+		timeStampsFull = append(timeStampsFull, timestamp.Format("2006-01-02 15:04:05")) //double check if the formatting is ok
 	}
+	fmt.Println(timeStampsFull)
 
 	// Turn map into sorted array of frequencies by timestamp
 	sortedFrequencies := GetSortedFrequencies(lengthMap)
 
+	//the file names will contain all the timestamps for a given folder that was processed
 	fmt.Printf("Sorted Array of Frequencies: \n%+v\n", sortedFrequencies)
+	freqOutFileName := fmt.Sprintf("static_data/freqData/freqOutFile%s.txt", timeStampsFull)
```

Comment:
[https://pkg.go.dev/path/filepath](https://pkg.go.dev/path/filepath)

We should be using this package to create file paths. Its a little bit safer. than doing it the current way.